axelor-stock/src/main/resources/demo/fr/stock_freightCarrierMode.csv

```diff
@@ -1,7 +1,3 @@
 "importId";"name";"carrierPartner.importId";"shipmentMode.importId"
-1;"Colissimo";8002;1
-2;"Chronopost";8004;2
-3;"TNT Express";8001;2
-4;"FedEx Home International";8005;3
-5;"La poste Business international solution";8002;3
-6;"Relais colis";8003;4
+1;"DHL";8001;1

```

Comment:
As that are the carrier partners, we could go to only one carrier with generic name: "Livraison fournisseur"

Reply:
okay, updated